src/login/translation/defaultMessages/userProfile.js

```diff
@@ -758,7 +758,7 @@ export const userData = {
     <FormattedMessage
       id="phone_format"
       defaultMessage={`Invalid telephone number. It must be a valid Swedish number, or written
-                            using international notation, starting with '+' and followed by 10-20 digits.`}
+                            using international notation, starting with '+' and followed by 6-20 digits.`}
```

Comment:
is the stuff about international notation useful here if we are only allowing Swe numbers?